lib/v2/screens/settings/components/settings_card.dart

```diff
@@ -21,8 +20,7 @@ class SettingsCard extends StatelessWidget {
   /// Use if the second row is a widget
   final Widget descriptionWidget;
 
-  /// Route to which the card navigates.
-  final String route;
+  final GestureTapCallback onTap;
```

Comment:
haha i think at some point i told you it was better to pass the on tap than to pass a route :)